process_manual/code_workflow.md

```diff
@@ -0,0 +1,244 @@
+the most if any conflicts occur.
+
+### Comments in the code
+
+C\# facilitates functionality to easily add a description to a function,
+class, etc., as seen below:
+
+``` 
+    /// <summary>
+    /// Description of a class e.g its functionality
+    /// </summary>
+    public class CodeConventionExample
+    {
+        /// <summary>
+        /// Description of the method that describes the behaviour
+        /// of the method?
+        ///(e.g. sum method: adds two numbers together)
+        /// </summary>
+        /// <param name="exampleParameter">Some parameter
+        ///(e.g. sum method: first operand)</param>
+        public void ExampleFunction(int exampleParameter)
+        {
+            //Insert code here
+        }
+      }
+```
+
+### Naming Conventions
+
+For the purpose of C\# programming, we recommend this naming scheme.
+
+    //Namespace name                       Interface name
+    namespace Path.CodeConventionExample : IExample
+    {
+        //Class name
+        public class CodeConventionExample
+        {
+            //Delegate name
+            delegate void MyDelegate(int parameterName);
+    
+            //Event name
+            event MyDelegate MyEvent;
+    
+            //Constant variable
+            const string ShipType = "DropShip";
+    
+            //Local variable
+            public int myTurn;
+    
+            //Field name (Class field)
+            private int _myVar;
+    
+            //Property name
+            public int MyProperty { get; set; }
+            
+            //Constructor name
+            public CodeConventionExample() {}        
+            
+            //Method name
+            public void ExampleFunction(int exampleParameter) {}
+        }
+    }
+
+### Other Aspects
+
+  - variable, class, and method names should be self-explanatory.
+
+  - Use explicit types (i.e. you should use int rather than var).
+
+  - Curly brackets should be vertically aligned and curly brackets
+    should have a line for themselves.
+
+  - Variables should be declared in the top of the class file, followed
+    by constructor.
+
+  - Comments should be on their own line.
+
+  - Indentation should be in form of 1 tab (4
+    spaces).
+
+# Documentation Standard
+
+The documentation consists of an overview of the implemented components,
+their responsibility, and the communication between them.
+
+# Version Control
+
+This section is used to cover the Git Workflow design used in this
+project. For version control we use GitHub. For the Git Workflow design
+we use GitFlow Workflow. The essentials parts are covered below, the
+entire documentation is seen in [the GitHub guides](https://help.github.com/en/articles/creating-issue-templates-for-your-repository). We **highly recommend** that
```

Comment:
This is not the link that with the explanation of gitflow, it is instead a link for creating issue templates is cited which seems to be a mistake.

Correct gitflow link: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow